src/Mod/TechDraw/Gui/TaskDimension.ui

```diff
@@ -62,9 +62,6 @@
 If 'Equal Tolerance' is checked this is also
 the negated value for 'Under Tolerance'.</string>
           </property>
-          <property name="alignment">
-           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
-          </property>
```

Comment:
Why do you delete this property? I set this not long ago purposely when I remember correctly.

Reply:
Didn't intentionally touch anything in the Formatting groupbox.  I used Qt Designer to add the new groupbox for Lines at the bottom.  Is that relevant?

Do you edit the actual ui file?

Reply:
> Didn't intentionally touch anything in the Formatting groupbox.

This is not the GroupBox but the QuantitySpinBox. The Alignment setting looks for me in Qt's Designer as of Qt 5.12.2:

![designer_n8YpuDYalc](https://user-images.githubusercontent.com/1828501/151093961-e55b3def-4c52-4e48-a5b7-b3ae5bb1c26d.png)


If you don't see it, then you have not installed FreeCAD's designer plugin. I compiled the plugin by myself and described how this is done:
https://wiki.freecadweb.org/Compile_on_Windows#Qt_Designer_plugin

> Do you edit the actual ui file?

I only use the Qt Designer.